NotificationBanner/Classes/BaseNotificationBanner.swift

```diff
@@ -268,9 +288,7 @@ open class BaseNotificationBanner: UIView {
     }
 
     internal func spacerViewHeight() -> CGFloat {
-        return NotificationBannerUtilities.isNotchFeaturedIPhone()
-            && UIApplication.shared.statusBarOrientation.isPortrait
-            && (parentViewController?.navigationController?.isNavigationBarHidden ?? true) ? 40.0 : 10.0
+        return basicYOffset + 10
```

Comment:
Why `10.0`?

Reply:
it's a margin value, replaced with an existing var